File: android/app/src/main/java/com/wizeline/panamexicans/data/models/UserStatus.kt
package com.wizeline.panamexicans.data.models

import com.google.firebase.Timestamp

data class UserStatus(
    val firstName: String = "",
    val lastName: String = "",
    val id: String = "",
    val lat: Double = 0.0,
    val lon: Double = 0.0,
    val status: String = "",
    val updatedAt: Timestamp = Timestamp.now()
)

File: iOS/PanAmexicans/PanAmexicans/App/RideSessions/Models/UserStatus.swift
//
//  UserStatus.swift
//  PanAmexicans
//
//  Created by Karen Gonzalez Velazquez on 17/02/25.
//

import Foundation
import class FirebaseCore.Timestamp

struct UserStatus: Codable {
    var id: String
    var firstName: String
    var lastName: String
    var lat: Double
    var lon: Double
    var status: String
    var updatedAt: Timestamp = Timestamp()
}
